Hi!

bezumkin opened the pull request:
Hey. I look at this beautiful snippet no updates. So, I took it upon myself to make a new version. 

Take a look at the changes. In my opinion - so much easier and more convenient.

b03tz commented:
Looking good, can you explain a little more what you did...did anything of the usage change? Alot of files are deleted and added as a whole so can't really see the coding differences very easily. Let me know!

bezumkin commented:
I will try (sory for my english)
1. Changed css minifier. Now styles not damaged on minification
2. Add js minifier.
3. Removed plugin - it is not needed anymore
4. Added normal filenames to destination files.
5. New caching system. Snippet adds timestamp to filenames, and if it call uncached - timestamp will always different, and browser will load files each time. If uncached - timestamp will be the same, and browser will take files from its own cache.
   On clear site cache - cached snippet will be deleted and generate new timestamp on first run. That is why plugin is not needed.
   Also, on generating minified files script looking to date of modification of source files and do not regenerate minified files, if they don`t changed.
6. Added parameters to snippet and improved buid script.
7. Other little improvements.

NunoBentes commented:
Hi

Just some questions question - On your point 5 - Don´t you think that could be a good idea, if the timestamp change just on css/js update, even if the plugin is cached or uncached?? Don´t you think this could make more sense? For me, if a css file name change without doing any change on it, don´t make sense. :)

Another question, do we need to use timestamp on the css/js files? I think you should mantain this feature, but don´t you think that could be a good idea to create a feature to remove timestamp and maintain only the desired css/js filename?

Sorry about all this questions, but this is a really great plugin, and i just want to give some feedbacks in the way how the plugin is working. :)
